b0realsim/label_to_perm_cond.py

```diff
@@ -0,0 +1,90 @@
+import nibabel as nib
```

Comment:
This whole file duplicates most of the code in the label2chi.py. It would be better to reuse that code; there could be one file / function that would be called label_to_property, and add a flag depending on the property you want to use (eg chi, perm_cond, etc). That way you won't need to add new files for every other new map you'll want to create in the future (eg T2, T1, etc), you may just need to change one line or two in that one function.

The other reason this is a better approach is from a review side - since there are no tests and many similar lines, I would need to go line by line to check that you properly copied / didn't accidentally change a line.

Overall though, it looks like it should work the way you intend it to, but you would know best if it works or not.